tests/test_features/test_pca.py

```diff
@@ -188,3 +191,26 @@ def test_scale_true_3d_execption(self):
         with pytest.raises(ValueError, match=e):
             pca = PCADecomposition(**params)
             pca.fit(X)
+    
+    @mock.patch('yellowbrick.features.pca.plt.sca', autospec=True)
```

Comment:
I am unsure about this test, I will wait on a review from @rebeccabilbro

Reply:
Yes, the test looks good! Patching pyplot's set current axes method ([`sca`](https://matplotlib.org/api/_as_gen/matplotlib.pyplot.sca.html) ) ensures that when we make the mock on line 207 (which is used to called `scatter` and retrieve the `alpha`), matplotlib doesn't complain about it.